charts/lagoon-core/templates/k8up.secret.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: 'baas-repo-pw'
```

similar to above I think we should have
```suggestion
  name: "{{ include "lagoon-core.fullname" . }}-baas-repo-pw"
```